modules/client-configuration/pages/clients-rocky.adoc

```diff
@@ -21,7 +18,7 @@ You do not need to disable SELinux to register {rocky} clients to {productname}.
 
 Before you can register {rocky} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
 
-The architectures currently supported are: `x86_64` and `aarch64`.
+The architectures currently supported are: `x86_64` and `aarch64`, partially {ppc64le} and {s390x}.
```

Comment:
Why partially?

Reply:
Maybe, for clarity I should better write "On version 9, also {ppc64le} and {s390x} are supported."